blazor/message/getting-started.md

```diff
@@ -63,7 +63,7 @@ var builder = WebApplication.CreateBuilder(args);
 // Add services to the container.
 builder.Services.AddRazorPages();
 builder.Services.AddServerSideBlazor();
-builder.Services.AddSyncfusionBlazor();
+builder.Services.AddSyncfusionBlazor(options => { options.IgnoreScriptIsolation = true; });
```

Comment:
This changes is not available in development branch

Reply:
Reverted the changes.